components/asana/sources/new-team/new-team.mjs

```diff
@@ -0,0 +1,47 @@
+import asana from "../../asana.app.mjs";
+
+export default {
+  key: "asana-new-team",
+  type: "source",
+  name: "New Team",
+  description: "Emit new event for each task added to an organization.",
+  version: "0.0.2",
+  dedupe: "unique",
+  props: {
+    asana,
+    organization: {
+      label: "Organization",
+      description: "Gid of a organization.",
+      type: "string",
+      propDefinition: [
+        asana,
+        "organizations",
+      ],
+    },
+    db: "$.service.db",
+    http: {
+      type: "$.interface.http",
+      customResponse: true,
+    },
+    timer: {
+      type: "$.interface.timer",
+      default: {
+        intervalSeconds: 60 * 15,
+      },
+    },
```

Comment:
I think you can also create a webhook making use of [team resource type](https://developers.asana.com/docs/team) instead of polling the teams

Reply:
Yes, I tried, but don't worked, this is the reason for listing all tags to emit the teams.

I thing this is outdated in the API docs.